packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -19,7 +22,18 @@ export function cloudflare<T extends Record<string, WorkerOptions>>(
 	return {
 		name: 'vite-plugin-cloudflare',
 		config() {
+			const alias = Object.fromEntries(
+				Object.entries(preset.alias).filter(
+					(i) => !preset.external.includes(i[1]),
+				),
+			);
```

Comment:
Last thing is to make this "per-environment".
Long term solution is to get Vite to allow it to be configured on each environment.
Short term solution is to use some virtual module resolution hackery... incoming.